webhookValidationConfig.go

```diff
@@ -66,9 +67,9 @@ type TTLVConfig struct {
 	Now    func() time.Time
 }
 
-// BuildValidators translates the configuration into a list of validators to be run on the
-// webhook.
-func BuildValidators(config ValidatorConfig) (Validator, error) {
+// BuildValidURLFuncs translates the configuration into a list of ValidURLFuncs
+// to be run on the webhook.
+func BuildValidURLFuncs(config ValidatorConfig) ([]ValidURLFunc, error) {
```

Comment:
```suggestion
func buildValidURLFuncs(config ValidatorConfig) ([]ValidURLFunc, error) {
```

I don't think we need to export this.